modules/rdg/src/bcs/AEFVBC.C

```diff
@@ -55,16 +51,12 @@ AEFVBC::computeQpJacobian()
   // assembly the input vectors, which are
   //   the constant monomial from the current element
   //   and the unit vector normal to the face
-
   std::vector<Real> uvec1 = {_uc1[_qp]};
   std::vector<Real> dwave = {_normals[_qp](0), _normals[_qp](1), _normals[_qp](2)};
```

Comment:
As we have talked, "dwave" can be "RealGradient". But as for "uvec1", the length is up to how many variables we have in a system of equations. So it may only be type of "vector<Real>".